src/EventSubscriber/InputDefinition/AliasArgument.php

```diff
@@ -30,7 +30,9 @@ public static function getSubscribedEvents() {
    */
   public function onCreateApplication(CreateApplicationEvent $event) {
     // Provide the option for specifying platform aliases.
-    $event->getApplication()->getDefinition()->addArgument(new InputArgument('alias', InputArgument::OPTIONAL, 'Provide a platform alias for remote execution.'));
+    $event->getApplication()->getDefinition()
+      ->addArgument(new InputArgument('alias', InputArgument::OPTIONAL, 'Provide a target platform alias for remote execution.'))
+      ->addArgument(new InputArgument('source', InputArgument::OPTIONAL, 'Provide a source platform alias for remote execution.'));
```

Comment:
This is a little concerning. Ideally we'd add arguments based on what a command supports.

Reply:
Right, rather than a generalised solution I moved the argument into the commands configure() method.